examples/encoding/plot_continuous_encoder.py

```diff
@@ -0,0 +1,115 @@
+"""
+=============================
+Regression on continuous data
+=============================
+
+This demonstrates how encoding models can be fit with multiple continuous
+inputs. In this case, a encoding model is fit for one electrode, using the
+values of all other electrodes as inputs. These inputs could also be stimulus
+features, such as spectral features of sound.
+
+Estimation of receptive fields and modeling with continuous inputs
```

Comment:
spatio-temporal receptive fields (STRF)

(they acronym is likely to be searched on the website)